nanome/api/plugin_instance.py

```diff
@@ -205,6 +205,32 @@ def update_node(self, node):
         """
         self._network._send(_Messages.node_update, node)
 
+    def set_menu_transform(self, index, position, rotation, scale):
```

Comment:
Should this have defaults for rotation and scale? Do I need to send everything if I just want to position it?

Reply:
yes